### DISCUSSION TITLE
in aggregate can set custom?

### DISCUSSION THREAD

**Discussion by chanthavong** [hasura/graphql-engine#7582] [General]
example 
```gql
query {
 order_items(where:{trash:{_eq:false}}){
      aggregate{
        count
        sum{
          total: (qty*price)
        }
      }
    }
 }
```

**Comment by BenoitRanque** [comment]
This is not possible.
You could use [postgres generated columns](https://www.postgresql.org/docs/12/ddl-generated-columns.html) to add a line total, and run an aggregate on that.
